ephemeris/run_data_managers.py

```diff
@@ -33,10 +33,12 @@ def wait(gi, job):
 
 def run_dm(args):
     url = args.galaxy or DEFAULT_URL
-    gi = GalaxyInstance(url=url, email=args.user, password=args.password)
-
-    # should test valid connection
-    log.info("List of valid histories: %s" % gi.users.get_current_user())
+    if args.api_key != "":
+        gi = GalaxyInstance(url=url, key=args.api_key)
+    else:
+        gi = GalaxyInstance(url=url, email=args.user, password=args.password)
+        # should test valid connection
```

Comment:
Yes. But the test is specific for the current user. When using the master api key: there is no current user. 
Currently the master api key cannot run the data managers because it does not have admin rights. But I hope it will get special permission to run the data managers as well in the future.